.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,53 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What steps did you take and what happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please follow the guidelines described in https://github.com/kubernetes-sigs/cluster-api/blob/main/SECURITY_CONTACTS
+      placeholder: "A clear and concise description on how to REPRODUCE the bug."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
```

Comment:
This part is technically asked in the first question

Reply:
@vincepri do you mean right at the beginning? if that is the case, we would start asking about the repro steps before letting the reporter define a problem they are facing & what they expected. Technically should be fine, but it might make more sense to keep the current order as is.

Reply:
I think it's just that this feels the same as 
```
placeholder: "A clear and concise description on how to REPRODUCE the bug."
```

I guess the question is, is "What steps did you take and what happened?" the same as the steps needed to reproduce. It sounds very similar to me, but no strong opinion.

If we think it's the same I would drop the "repro" field

Reply:
@sbueringer @vincepri removed the repro field from the bug report template as suggested: https://github.com/kubernetes-sigs/cluster-api/pull/7819/commits/ec3f533f4c1533d933aebd73f46caae02977889f